src/adapters/mongoose.js

```diff
@@ -269,6 +277,12 @@ module.exports = (model, opts) => {
       schema.isRequired = isRequired;
     }
 
+    schema.isGenerated = getGenerated(fieldInfo);
+
+    if (getReadOnly(fieldInfo)) {
+      schema.isReadOnly = true;
+    }
```

Comment:
This seems to be another intent.
What you are trying to do is to forbid `_id` field edition whatever the use case, is that it?
It means that the field won't be configurable as "readonly" or not in the UI configuration right?

Reply:
I'm doing the same thing that we do with primary keys with `SQL` databases, and forbid the edition of ids, yes.

I'm not sure if the whole process of testing was working correctly if I did not include this change too. For what I remember, it did not.

What do you suggest me to do about this change?

Reply:
@arnaudbesnier ⬆️